docs/content/users/usage/troubleshooting.md

```diff
@@ -98,6 +98,8 @@ sudo apachectl stop
 
 Here are some of the other common processes that could be using ports 80/443 and methods to stop them.
 
+* macOS Content Filtering: Under "Screen Time" -> "Choose Screen Time content and privacy settings" turn off "Content and Privacy". Then you must reboot. This has been a common issue with macOS Sonoma.
+* macOS or Linux Homebrew: Look at `brew services` and temporarily `brew services stop` anything that's running.
```

Comment:
Run commands, and maybe further clarify what’s going on here. (TIL about `brew services`, and I _think_ I get the intent here.)

```suggestion
* macOS or Linux Homebrew: Look for active processes by running `brew services` and temporarily running `brew services stop` individually to see if it has any impact on the conflict.
```